api/v1alpha1/openapi.yaml

```diff
@@ -4628,6 +4628,8 @@ components:
           $ref: '#/components/schemas/ImageVolumeSource'
       required:
         - image
+      not:
```

Comment:
Note: required to allow the openapi validator to determine the difference between the three types of

```yaml
- name: one
  mount:
    path: /path
- name: two
  image:
    reference: some_ref
- name: three
  image:
    reference: some_ref
  mount:
    path: /path2
```